src/composite_algs.jl

```diff
@@ -15,7 +15,7 @@ function is_stiff(integrator, alg, ntol, stol, is_stiffalg)
     stiffness = abs(eigen_est * dt / alg_stability_size(alg)) # `abs` here is just for safety
     tol = is_stiffalg ? stol : ntol
     os = oneunit(stiffness)
-    bool = stiffness > os * tol
+    bool = !(stiffness <= os * tol)
```

Comment:
why this change?

Reply:
in case `eigen_est` is `NaN`. It's a pretty good sign that things are going wrong, but since it's an estimate, if the estimate is `NaN` we really shouldn't be using an explicit solver.